aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -328,6 +328,38 @@ void add_dense_sparse_worker_cpu(Tensor& r, Scalar value, const SparseTensor& sp
   });
 }
 
+template <typename scalar_t>
+void add_dense_sparse_worker_cpu(Tensor& r, Scalar value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
+
+  int64_t values_dense_size = values.stride(0);
```

Comment:
If values is 1d contiguous, it is always == 1.

And that is the only reason why invalid `for` loop, not corrupting your data
```cpp
      for (auto v = 0; v < values_dense_size; v++) {
        r_ptr[r_index + v] += cast_value * v_ptr[v_index + v];
      }
```

Reply:
Please remove this loop

Reply:
This is to get the dense dimension element numbers of hybrid sparse tensor

Reply:
For hybrid sparse tensor, the values can be like below
>>> i = torch.LongTensor([[2, 4]])
>>> v = torch.FloatTensor([[1, 3], [5, 7]])
The values.stride(0) should get 2 here

Reply:
Right, thank you for pointing it out. I will check again with hybrid sparse in mind.